### PR TITLE
Integrate Codacy with project.

### PR DESCRIPTION
**Pull request by sidhyatikku**
I've integrated codacy, added .codacy.yml & added codacy badge to readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+![Build Status](https://travis-ci.org/sidhyatikku/lead-management-server.svg?branch=master)
```
Comment (m-murad) [inline README.md]: Update once the PR is merged

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/03deaa8060f447acb862108079f89d35)](https://www.codacy.com/app/sidhyatikku/lead-management-server?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=sidhyatikku/lead-management-server&amp;utm_campaign=Badge_Grade)
```
Comment (m-murad) [inline README.md]: Update once the PR is merged

**Comment by m-murad** [comment]
Please resolve the conflict

**Comment by sidhyatikku** [comment]
I've deleted Codacy status icon from readme.md but you'll need to add it

**Pull request merged**